### DISCUSSION TITLE
How to add `rust` filetype for the tailwindcss lsp ?

### DISCUSSION THREAD

**Discussion by leo91000** [VonHeikemen/lsp-zero.nvim#136] [Q&A]
Hello !

I am trying to make the tailwindcss lsp start for rust files. I tried to do this

```lua
local lsp = require('lsp-zero')
lsp.preset('recommended')
lsp.ensure_installed({
    'tailwindcss',
})
lsp.configure('tailwindcss', {
    filetypes = {
        "html",
        "javascriptreact",
        "typescriptreact",
        "rescript",
        "vue",
        "svelte",
        "rust"
    },
})
lsp.setup()
```

However, when I run the `:LspInfo` on a rust file inside a rust project (with a `tailwind.config.js` file), the tailwindcss lsp doesn't seems to be running.

Am I doing something wrong ?
What could I do to debug the issue ?

**Comment by VonHeikemen** [comment]
That looks about right. Does `:LspInfo` shows you a list of "other clients"? It should begin with a text like this:

```
Other clients that match the filetype: rust
.....
```

**Comment by hmuendel** [comment]
The above config almost worked for me, but I had to tell the tailwind lsp how to handle rust files with a language from [here](https://github.com/tailwindlabs/tailwindcss-intellisense/blob/master/packages/tailwindcss-language-service/src/util/languages.ts)

 Something like this seem to work:

```lua
lsp.configure('tailwindcss', {
  init_options = {
    userLanguages = {
      rust = "html",
    }
  },
  filetypes = {
    "html",
    "javascriptreact",
    "typescriptreact",
    "rescript",
    "vue",
    "svelte",
    "rust"
  },
})

```
